chpl/chpl-api/e2e/data/2015chplids-test-data.json

```diff
@@ -0,0 +1,18 @@
+[
+ {
+  "certifiedProductId":9437,
+  "year.testingLab.certBody.vendorCode.productCode.versionCode.icsCode.addlSoftwareCode.certDateCode":"15.04.04.2214.Proc.14.00.0.180427"
```

Comment:
Two data files 2014 and 2015 can be combined by keeping unique variable names.

Reply:
This is done